tests/Integration/Db/ContextChat/TaskMapperTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * SPDX-FileCopyrightText: 2021 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+namespace OCA\Mail\Tests\Integration\Db;
+
+use ChristophWurst\Nextcloud\Testing\DatabaseTransaction;
+use ChristophWurst\Nextcloud\Testing\TestCase;
+use OCA\Mail\Db\ContextChat\TaskMapper;
+use OCP\DB\QueryBuilder\IQueryBuilder;
+use OCP\IDBConnection;
+
+class TaskMapperTest extends TestCase {
+	use DatabaseTransaction;
+
+	/** @var IDBConnection */
+	private $db;
+
+	private TaskMapper $mapper;
+
+	protected function setUp(): void {
+		parent::setUp();
+
+		$this->db = \OC::$server->getDatabaseConnection();
```

Comment:
nit: ocp\server::get(Idbconnection)? then no private API is needed

Reply:
Good point!